yw_website/yw_rest_services/tests.py

```diff
@@ -4,27 +4,26 @@
 from yw_db.models import Workflow
 import uuid
 
-# Create your tests here.
 class YwSaveTestCase(TestCase):
     def setUp(self):
         self.client = Client()
+        self.username = str(uuid.uuid1())
+        self.user = User.objects.create_user(username=self.username, password='12345')
```

Comment:
Also a good addition

Reply:
@aniehuser added workflow_id to path for updating a workflow.

Reply:
@aniehuser you can check now